service/actions/websocket.js

```diff
@@ -224,7 +224,7 @@ async function handleNewPostEvent(msg, dispatch, getState) {
         }
     ]), getState);
 
-    if (userId === users.currentId || post.channel_id === currentChannelId || isSystemMessage(post)) {
+    if ((userId === users.currentId && !isSystemMessage(post)) || post.channel_id === currentChannelId || shouldIgnorePost(post)) {
```

Comment:
@enahum RE: my first comment, our logic doesn't match the webapp behavior:
In the webapp, when a user receives an ignored message in an unread channel it doesn't mark it as read.

RE: my second comment, I meant when the user does the action on one device, the channel will be marked unread on other devices. It's not a significant issue but I think we can avoid it.